common/.golangci.yml

```diff
@@ -87,3 +87,7 @@ linters:
           - contextcheck
         path: libimage
         text: LookupImage
+
+issues:
```

Comment:
+1 I did this in podman a long time ago, no idea why they choose such default really
https://github.com/containers/podman/commit/93c35a703876063e181c18d9239a1bd5c7dcc578

It always just leads to the wrong assumption of let me just fix these three issues really quick.